mmcv/ops/csrc/pytorch/cpu/bezier_align.cpp

```diff
@@ -0,0 +1,567 @@
+            output.data_ptr<scalar_t>(), aligned_height, aligned_width,
+            static_cast<scalar_t>(spatial_scale), sampling_ratio,
+            aligned, channels, height, width); });
+}
+
+void BezierAlignBackwardCPULauncher(Tensor grad_output, Tensor rois,
+                                    Tensor grad_input, int aligned_height,
+                                    int aligned_width, float spatial_scale,
+                                    int sampling_ratio,
+                                    bool aligned)
+{
+  int output_size = grad_output.numel();
+  int channels = grad_input.size(1);
+  int height = grad_input.size(2);
+  int width = grad_input.size(3);
+
+  // get stride values to ensure indexing into gradients is correct.
+  int n_stride = grad_output.stride(0);
+  int c_stride = grad_output.stride(1);
+  int h_stride = grad_output.stride(2);
+  int w_stride = grad_output.stride(3);
+
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(
+      grad_output.scalar_type(), "BezierAlign_backward", [&]
+      { BezierAlignBackward<scalar_t>(
+            output_size, grad_output.data_ptr<scalar_t>(),
+            rois.data_ptr<scalar_t>(), grad_input.data_ptr<scalar_t>(),
+            aligned_height, aligned_width, static_cast<scalar_t>(spatial_scale),
+            sampling_ratio, aligned, channels, height, width,
+            n_stride, c_stride, h_stride, w_stride); });
+}
+
+void bezier_align_forward_cpu(Tensor input, Tensor rois, Tensor output,
+                              int aligned_height,
+                              int aligned_width, float spatial_scale,
+                              int sampling_ratio, bool aligned)
+{
+  BezierAlignForwardCPULauncher(input, rois, output,
+                                aligned_height, aligned_width, spatial_scale,
+                                sampling_ratio, aligned);
+}
+
+void bezier_align_backward_cpu(Tensor grad_output, Tensor rois, Tensor grad_input,
+                               int aligned_height, int aligned_width,
+                               float spatial_scale, int sampling_ratio,
+                               bool aligned)
+{
+  BezierAlignBackwardCPULauncher(grad_output, rois, grad_input,
+                                 aligned_height, aligned_width, spatial_scale,
+                                 sampling_ratio, aligned);
+}
+
+void bezier_align_forward_impl(Tensor input, Tensor rois, Tensor output,
+                               int aligned_height, int aligned_width,
+                               float spatial_scale, int sampling_ratio,
+                               bool aligned);
+
+void bezier_align_backward_impl(Tensor grad_output, Tensor rois, Tensor grad_input,
+                                int aligned_height, int aligned_width,
+                                float spatial_scale, int sampling_ratio,
+                                bool aligned);
+
+REGISTER_DEVICE_IMPL(bezier_align_forward_impl, CPU, bezier_align_forward_cpu);
```

Comment:
`bezier_align_forward_cpu` and `bezier_align_backward_cpu` are note necessary, register with `BezierAlignForwardCPULauncher` and `BezierAlignBackwardCPULauncher`